tests/unit/test_vmware_exporter.py

```diff
@@ -116,6 +119,34 @@ def test_collect_vms():
     }
     assert metrics['vmware_vm_guest_disk_capacity'].samples[0][2] == 100
 
+    # VM tools info (vmguest)
+    assert metrics['vmware_vm_guest_tools_running_status'].samples[0][1] == {
+        'vm_name': 'vm-1',
+        'host_name': 'host-1',
+        'cluster_name': 'cluster-1',
+        'dc_name': 'dc',
+        'tools_status': 'toolsOk',
```

Comment:
maybe just call this label `status`?

Reply:
I say just have it match the api. I don't like interpreting external apis in our own way. Creates a level of abstraction thats is not needed.